packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js

```diff
@@ -377,10 +377,10 @@ export default class DatabaseService {
       .then(({ data }) => data);
   }
 
-  editUser(projectId, engine, databaseId, user) {
+  editUser(projectId, engine, databaseId, userId, user) {
```

Comment:
```suggestion
  editUser(projectId, engine, databaseId, userId) {
```

Reply:
We need the `user` variable which contains the new data of the user having `userId` for id.
However, we don't want to put the `userId` as a property of `user` because the api does not allow changing `id` on the `PUT` method